### DISCUSSION TITLE
any way to group color versions of certain traits?

### DISCUSSION THREAD

**Discussion by BakeSloth** [nftchef/art-engine#173] [Help + Q&A] [answered]
I have different colors of all my traits, and would like to make sure that the first color chosen as a background also chooses the same color of other traits, still using the weights assigned.

example:
-Background
--Clouds#50
---Clouds Blue#25.png
---Clouds Red#25.png
---Clouds Green#25.png
---Clouds Purple#25.png
--Stars#50
---Stars Blue#25.png
---Stars Red#25.png
---Stars Green#25.png
---Stars Purple#25.png

-Body
--Male#50
---Male Blue#25.png
---Male Red#25.png
---Male Green#25.png
---Male Purple#25.png
--Female#50
---Female Blue#25.png
---Female Red#25.png
---Female Green#25.png
---Female Purple#25.png

if "Clouds Purple.png" is chosen first in the stack,  I want all of the following traits to also be purple, but still keeping the 50/50 male to female weight percentages in play. 

I don't want to force specific traits, but I want to force specific colors in the file names to take priority based on subfolder weight.

Any suggestions would be greatly appreciated!

**Comment by neilryanh** [comment, marked as answer]
Does this thread help? Seems similar, but maybe not.
https://github.com/nftchef/art-engine/issues/172

**Reply by BakeSloth** [reply]
Worked a charm. Thanks!
